From 5571f50fd9d53b98d7246e9d208c5fd5bc643684 Mon Sep 17 00:00:00 2001
From: chandavath <rcrajagopal@habana.ai>
Date: Fri, 01 Mar 2024 14:42:38 +0200
Subject: [PATCH] [SW-137122] gaudi3: erom rec: fix pldm build fail

Fixed pldm build failure. in pldm build, it could find the definition
for this arcboot_rom_get_arc_addr function. so included exports.elf
in pldm Makefile.

Change-Id: Ifa10906fe8553afda7a07eae5d67fd684a2c1862
Signed-off-by: chandavath <rcrajagopal@habana.ai>
---

diff --git a/app/pldm_bloader/Makefile b/app/pldm_bloader/Makefile
index fd2de27..9823a37 100644
--- a/app/pldm_bloader/Makefile
+++ b/app/pldm_bloader/Makefile
@@ -26,6 +26,7 @@
 PLDM_BLOADER_LD_SCRIPT := $(shell ls $(PREBOOT_ROOT_DIR)/$(HL_DEV_ARCH)/pldm_bloader.lds.S)
 LDFLAGS := -g -nostdlib -static
 LDFLAGS += -Wl,--gc-sections
+LDFLAGS += -Wl,--just-symbols $(shell ls $(PREBOOT_ROOT_DIR)/$(HL_DEV_ARCH)/exports.elf)
 LDLIBS := -lgcc
 
 PLDM_BLOADER_SRCS_C := $(shell ls *.c)
